### ISSUE TITLE
Any plans on updating this amazing mod to 1.14?

### ISSUE THREAD

**Issue by 123Kappa** [MinimallyCorrect/TickProfiler#112]
Any plans on updating this amazing mod to 1.14?

**Comment by LunNova** [comment]
Yes, but can't give a time scale, sorry. Been working on some behind the scenes stuff for dependencies involved in patching.

Want to make it compatible with fabric and forge, just to add complications. :c

**Comment by miiichael** [comment]
Should I assume you'll be targetting 1.15 now?
